src/index.html

```diff
@@ -7,6 +7,41 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar">
+      <div class="month month--start--day_sun month--month--length-29">
+        <div class="days"></div>
```

Comment:
Use BEM, and add modifiers to calendar, like it is mentioned in the task's description